py/picca/fitter2/data.py

```diff
@@ -313,8 +317,7 @@ def __init__(self,dic_init):
             hmet.close()
 
     def xi_model(self, k, pk_lin, pars):
```

Comment:
why did you modify?

Reply:
I don't reminder changing something here ...